.github/workflows/tarball.yml

```diff
@@ -0,0 +1,15 @@
+name: Tarball evaluation checks
+
+permissions: read-all
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v13
+    - run: nix-build pkgs/top-level/release.nix -A tarball --arg supportedSystems '[ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ]'
```

Comment:
Nit: maybe at least move it to the first place?  (Otherwise it takes quite some time to get to the error specific for aarch64-darwin, and we don't check it anywhere else so far.)

Reply:
OOMs with all anyway:
https://github.com/NixOS/nixpkgs/pull/129427/checks?check_run_id=2998217938 https://github.com/NixOS/nixpkgs/pull/129427/checks?check_run_id=2998218312

Reply:
I've pushed a fix that make swap partition 8GB. :crossed_fingers:

Reply:
I just checked the runtime of the checks and so far they are not done after 67 minutes which is a bit on the long side.